ulakbus/services/common/document_render.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from zato.server.service import Service
+
+
+class RenderDocuments(Service):
```

Comment:
Add documentation for your module, objects and methods. see this one: https://github.com/zetaops/ulakbus/blob/develop/ulakbus/services/ogrenci/sinav_plani.py#L26